Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -108,6 +107,11 @@ private void IgniteOnCollide(EntityUid uid, IgniteOnCollideComponent component,
             if (!EntityManager.TryGetComponent(otherEnt, out FlammableComponent? flammable))
                 return;
 
+            if (args.OurFixtureId == SharedFlyBySoundSystem.FlyByFixture)
+            {
+                return;
```

Comment:
just add a fixtureId to IgniteOnCollideComponent and check that the fixture equals our fixture rather than specifically checking this one. Entities can have any amount of fixtures and we should be able to tune this to apply to only one in particular. This will probably require having to update it for entities with IgniteOnCollide.

Reply:
I did what you suggested, but for projectiles I have to either make the fixture bigger than the projectile one, making the projectiles ignite while missing again, or the projectile collision will sometimes happen first. 
If the projectile collides first, deleteOnCollide will delete the projectile before it collides with the ignition fixture.
 This is why I made it ignite on projectile fixture collision or on the new ignition fixture collision(currently only used for the magic rune but can be used for anything else too).